change/@internal-react-components-8b40a12e-a568-41b7-85f8-0021a25489f6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Mobile Participant pane now spans full sidepane width",
```

Comment:
nit: this actually impacts the ParticipantItem public component we export. Cna you gear this change file to be what changed in this package. e.g. `Removed 20rem max-width restriction on ParticipantList component. Max-width is now set to 100%`

Reply:
I've tried different types of combinations, but without setting a width the participant item overflows off screen when names are very long.
<img src="https://user-images.githubusercontent.com/97130533/156813515-277b75a1-5f20-43c3-87d5-3d0867d271e7.png" width="30%" height="30%"/>

Also found that ellipsis has a couple of restraints for it to work:
```
text-overflow:ellipsis; only works when the following are true:

The element's width must be constrained in px (pixels). Width in % (percentage) won't work.
The element must have overflow:hidden and white-space:nowrap set.
```

Reply:
You should be able to use `max-width: 100%`, the problem with `max-width: 100%` is each parent element will also need this constraint - i.e. what is the 100%? if its 100% of a div that can be any width, then it can be any width. If you walk up the DOM tree and continue to give each parent element `max-width: 100%` I imagine this will eventually give you what you want.
Let me know if you want to pair on this!

Reply:
Thank you James!!
I put 100% max-width on all divs going up one by one by one and then noticed that one specific stack parent div was the key.
<img src="https://user-images.githubusercontent.com/97130533/156819118-7d4df542-05a6-46de-bc8f-22809e4e9b45.png" width="30%" height="30%"/>